client/header/activity-panel/panels/stock/card.js

```diff
@@ -79,6 +86,9 @@ class ProductStockCard extends Component {
 		this.setState( { editing: false, edited: true } );
 
 		updateProductStock( product, quantity );
+		this.recordStockEvent( 'save', {
+			quantity,
```

Comment:
Maybe more of a question for @pmcpinto - what is the value/use of knowing the stock quantity being modified? Without any context it seems rather useless.. and to be clear I'm not advocating for adding more data to this event.